src/charm.pyi

```diff
@@ -1,5 +1,5 @@
 # Copyright 2022 Canonical Ltd.
```

Comment:
Do the type hints have to be updated with the addition of a new function?

Reply:
I have updated type hints.